nexus/src/app/vpc.rs

```diff
@@ -7,6 +7,7 @@
 use crate::authz;
 use crate::context::OpContext;
 use crate::db;
+use crate::db::identity::{Asset, Resource};
```

Comment:
We've been trying to move to item-per-line imports to aid conflict resolution.

Reply:
There are a couple of other places below with grouped imports as well, lines 32 and 36.

Reply:
Sure thing, fixed in `f3b64ae0`.